tests/inventory/pipeline_builder_test.py

```diff
@@ -0,0 +1,170 @@
+# Copyright 2017 Google Inc.
```

Comment:
Hmmm, not unless it is extracted into a common util function.   The way it is here, it's got functional test coverage to make sure it is working correctly.  :smile: